opendebates/utils.py

```diff
@@ -133,7 +134,11 @@ def sort_list(citations_only, sort, ideas):
 
 
 def get_site_mode():
-    return SiteMode.objects.get_or_create()[0]
+    try:
+        return SiteMode.objects.get()
+    except SiteMode.DoesNotExist:
+        with readwrite_db():
+            return SiteMode.objects.get_or_create()[0]
```

Comment:
We could be in a race with another process or server trying to do this at the same time.